.github/workflows/ci.yml

```diff
@@ -30,83 +30,83 @@ jobs:
         with:
           python-version: "3.x"
       - run: pip install check-manifest && check-manifest
-  
+
   # Check tests pass on multiple Python and OS combinations
   test:
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9, "3.10", "3.11"]
-        os: [ubuntu-latest, macOS-latest, windows-latest]
+        python-version: [3.9, "3.10", "3.11", "3.12"]
```

Comment:
nit. I believe the quotations are only necessary for versions with a trailing zero?

Reply:
I can remove em - I was following the pre-existing pattern which already had 3.11 in quotes.